src/utils/utils.ts

```diff
@@ -7,22 +13,15 @@
  * @param includeDate - Whether the date should be displayed, defaults to true
  * @param includeTime - Whether the time should be displayed, defaults to true
  */
+export const createDateTime = ({ date, includeDate = true, includeTime = true }: ICreateDateTimeProps): string | null => {
```

Comment:
I will keep `includeDate` and `includeTime` parameters, because this simplifies not including either date or time. But object will be returned, which will include date, time and datetime (and only datetime will be affected by those two parameters).